wgs_somatic-run-wrapper.py

```diff
@@ -131,16 +133,35 @@ def wrapper():
                     n = k
                     n_ID = [val for val in pair_dict_all_pairs.get(k) if val != 'normal'][0]
                     if n_ID == t_ID or t_ID == n.split("DNA")[1] or n_ID == t.split("DNA")[1]: # if we change to pair id instead of tumorNormalID this is needed
```

Comment:
Could this be written a bit cleaner? Difficult to know at a glance what's going on.

Reply:
added some comments about it in commit @ 0a04b59.
But should probably be solved in a nicer way later